Contact.html

```diff
@@ -0,0 +1,164 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>html200-adopt-a-dog</title>
+    <link rel="stylesheet" href="./styles/styles.css">
+  </head>
+  <body>
+
+
+    <header class="header-images">
+      <div class="background2Box">
```

Comment:
Choose a more semantic name for this class. background2Box doesn't mean much and is hard to read. something like logo-container or something